content/blog/eng/v-protetto17-5-2024.md

```diff
@@ -0,0 +1,120 @@
+---
+
+authors: ["mgarza", "ana-radujko"]
+comments: true
+date: "2024-03-11 17:00:00"
+draft: false
+share: true
+categories: ['Protected Fridays']
+title: "Venerdì Protetto | May 2024 edition"
+description: "What happened in the last issue of Venerdì Protetto?"
+languageCode: "en-US"
+type: "post"
+twitterImage: '/images/social/social-preview-vp.png'
+twitterLargeImage: '/images/social/social-preview-vp.png'
+toc: false
+
+---
+
+
+Topics:
+
+
+During the last Venerdì Protetto (held on May 17, 2024), we had both a talk and a workshop. There they are:
+
+- [Hack the Box Workshop](#hack-the-box-workshop) by [Alessio Giorgianni](https://www.linkedin.com/in/alessio-giorgianni-b90500123/) (Facile.it)
+- [How to develop cognitive flexibility?](#how-to-develop-cognitive-flexibility) by [Arduino Mancini](https://www.linkedin.com/in/arduinomancini/) ([TIBICON](https://www.tibicon.net/))
+
+
+# Hack the box Workshop
+
+### What is this workshop? 
+After the successful experience of [having a code challenge in March 2024](https://engineering.facile.it/blog/eng/v-protetto8-3-2024/ "Previously, on Venerdì Protetto... a code challenge!"), we decided to go further in delivering experiences to our colleagues.
+
+Scouting around, I asked help to Alessio Giorgianni, a developer with a passion of white hat hacking competition. We agree upon trying an experience using a platform, called [Hack the Box](https://academy.hackthebox.com/). Hack the Box Academy offers lots of formation about IT Security and, in our case, some exercises we can use for hacking dummy application. We agree upon using a non-trivial quests, called [Jerrytok](https://www.hackthebox.com/achievement/challenge/48545/638). Jerrytok is a simple web-application, written in PHP, undisclosing the harmness of using template engines in not-proper way. It's a good introduction to SSTI, [Server Side Template Injection](https://portswigger.net/web-security/server-side-template-injection).
+
+![Please wear your favorite hacker hoodie before trying the workshop]( /static/images/vp-may-2024/hack-the-box-v0-a56fw7h8a2aa1.webp "Hack the Box Wallpaper")
+*Please wear your favorite hacker hoodie before trying the workshop*
+
+
+### Preparing the workshop
+
+
+![Topics of the workshop](/static/images/vp-may-2024/topics.png "Topics of the workshop")
+*Topics we focused on*
+
+Let's not hide ourself behind a finger: Hacking an application is not a trivial task. It requires a fond knowledge of how application works, several tools are used to diagnose and to try to break the application. But preparing all this stuff requires quite a lot of time.
```

Comment:
- **ourselves** In questo caso sostituirei l'espressione Let's not hide ourselves beside a finger, che mi sembra più italiana che inglese e scrivere, con un Let's not fool ourselves. 
- ...knowledge of how **the** application works...
- ...works, **while** we must use several tools to diagnose...
- Io toglierei quel But dall'inizio della frase perché mi sembra che non stia in contrasto con la frase precedente.